includes/Modules/AdSense.php

```diff
@@ -421,6 +422,24 @@ protected function parse_data_response( Data_Request $data, $response ) {
 		return parent::parse_data_response( $data, $response );
 	}
 
+	/**
+	 * Checks for the state of an Account, whether close or not. 
+	 * 
+	 * @since n.e.x.t
+	 * 
+	 * @param object $account The AdSense account object.
+	 * 
+	 * @return bool
+	 */
+	public static function is_account_not_closed($account){
+
+		if( $account->getState() === 'CLOSED' ) {
+			return true;
+		}
+
+		return false;
```

Comment:
This condition has to be the opposite. Also, we can simplify it to a single line.
```suggestion
		return 'CLOSED' !== $account->getState();
```